packaging/os/openbsd_pkg.py

```diff
@@ -18,8 +18,10 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+import os
 import re
 import shlex
+import sqlite3
```

Comment:
so this is a new requirement and should be documented, as it is not present in python 2.4